subsystems/mvdm/ntvdm/vddsup.c

```diff
@@ -502,8 +502,6 @@ VOID VDDResumeUserHook(VOID)
     }
 }
 
-
-
```

Comment:
```suggestion


```
There is a reason for the spacing

Reply:
Reverted, though I wonder what that reason can be.
Suggest an explicit separating comment?